plugins/python/redisgears_python.c

```diff
@@ -4737,39 +4833,7 @@ static void RedisGearsPy_InnerExecute(RedisModuleCtx* rctx, BackgroundDepsInstal
 
     ptctx->createdExecution = NULL;
     ptctx->currentCtx = NULL;
-}
-
-static void RedisGears_OnRequirementInstallationDone(ExecutionPlan* ep, void* privateData){
-    BackgroundDepsInstallCtx* bdiCtx = privateData;
-
-    RedisModuleCtx* rctx = RedisModule_GetThreadSafeContext(bdiCtx->bc);
-
-    size_t nErrors = RedisGears_GetErrorsLen(ep);
-
-    if(nErrors > 0){
-        Record* errorRecord = RedisGears_GetError(ep, 0);
-        size_t errorLen;
-        const char* errorStr = RedisGears_StringRecordGet(errorRecord, &errorLen);
-        RedisModule_ReplyWithError(rctx, errorStr);
-        RedisGears_SessionRegisterCtxFree(bdiCtx->session->srctx);
-        bdiCtx->session->srctx = NULL;
-    }else{
-        bdiCtx->session->isInstallationNeeded = false;
-        RedisGearsPy_InnerExecute(rctx, bdiCtx);
-    }
-
-    RedisGears_DropExecution(ep);
-
-    // free bdiCtx
-    PythonSessionCtx_Free(bdiCtx->session);
-    if (bdiCtx->oldSession) PythonSessionCtx_Free(bdiCtx->oldSession);
-    RG_FREE(bdiCtx->script);
-    if(bdiCtx->bc){
-        RedisModule_UnblockClient(bdiCtx->bc, NULL);
-    }
-    RG_FREE(bdiCtx);
-
-    RedisModule_FreeThreadSafeContext(rctx);
+    return REDISMODULE_OK;
```

Comment:
Curious - why didn't you return a REDISMODULE_OK before? Just missing? This is for my own education.